### ISSUE TITLE
Update copy on filter modal

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#826]
As part of our design pass coming out of Accessibility feedback, we've modified the copy on the filter modal to make it more readable. The checkbox at the bottom that says 'Include Results with Incomplete Information' should now read 'Show homes with missing information'.
